sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/ApiRequest.kt

```diff
@@ -76,6 +76,11 @@ interface ApiRequest {
      */
     val responseCode: Int?
 
+    /**
+     * HTTP Response Headers
+     */
+    val responseHeaders: Map<String, List<String>>?
```

Comment:
nullable vs empty map?

Reply:
id lean towards empty map

Reply:
I was copying the pattern set by `responseCode`, of being null till request has been sent. But empty map would be fine for this, and less null checks